packages/atlas/src/components/_inputs/FileSelect/FileSelect.tsx

```diff
@@ -16,24 +16,22 @@ import { FileType } from '@/types/files'
 
 import {
   ButtonsGroup,
+  Content,
   DragAndDropArea,
   DragDropText,
+  FileHoverOverlay,
+  FileSelectedOverlay,
   InnerContainer,
-  SelectedFileInfo,
-  SelectedFileInfoBackground,
-  SelectedFileInfoHeading,
-  SelectedFileInfoInnerContainer,
   Thumbnail,
   Title,
 } from './FileSelect.styles'
 
 export type FileSelectProps = {
-  fileType: FileType
+  type: 'video-thumbnail' | 'playlist-thumbnail' | 'video-file'
```

Comment:
Why do you resign from `FileType`? Can't we extend it to include playlist thumbnail?

Reply:
Because playlist thumbnail it's not a `fileType` per se, that would make it confusing because there are a lot of differences between each type, and it doesn't just come to what type of file is accepted